frontend/src/store/slices/accessRequests/useAccessRequests.ts

```diff
@@ -1,31 +1,36 @@
 import { useAppDispatch } from './../../hooks';
 import { showLoading, hideLoading } from 'react-redux-loading-bar';
-import CustomAxios from 'customAxios';
-import { ENVIRONMENT } from 'constants/environment';
 import { AxiosResponse, AxiosError } from 'axios';
 import { IAccessRequest } from 'interfaces';
 import * as actionTypes from 'constants/actionTypes';
 import * as API from 'constants/API';
 import React from 'react';
+import { useApiAccessRequests } from './../../../hooks/pims-api/useApiAccessRequests';
```

Comment:
I think you can just go `hooks/pims-api`

Reply:
Good point, I have this in a follow up PR.